RetailCoder.VBE/UnitTesting/UnitTestHelpers.cs

```diff
@@ -77,5 +77,12 @@ private static IEnumerable<Declaration> GetTestModuleProcedures(RubberduckParser
                         item.ParentDeclaration.DeclarationType == DeclarationType.ProceduralModule &&
                         item.ParentDeclaration.Annotations.Any(a => a.AnnotationType == AnnotationType.TestModule));
         }
+
+        public static IEnumerable<Declaration> GetTestModules(this RubberduckParserState state)
+        {
+            return state.AllUserDeclarations.Where(item =>
+                        item.DeclarationType == DeclarationType.ProceduralModule &&
+                        item.Annotations.Any(a => a.AnnotationType == AnnotationType.TestModule));
+        }
     }
```

Comment:
I would rename this file to `UnitTestingExtensions`. IDK, there's something about `Helpers`...